MediaRecorder.html

```diff
@@ -326,11 +329,11 @@
             playable, but the combination of all the Blobs from a completed
             recording MUST be playable.</p>
 
-            <p> If any Track within the MediaStream is muted at any time (i.e.,
-            if its <code>muted</code> flag is set to <code>true</code>), the UA
-            MUST insert black frames or silence until the Track is unmuted, or
-            record nothing, depending on the value of
-            <a>ignoreMutedMedia</a>.</p>
+            <p> The UA MUST insert black frames or silence, or record nothing,
+            depending on the value of <a>ignoreMutedMedia</a>, for as long as
+            a given <var>track</var> within the MediaStream  is
+            <a href="https://www.w3.org/TR/mediacapture-streams/#widl-MediaStreamTrack-muted">
+            muted</a>. This behaviour only applies to the muted tracks.</p>
 
```

Comment:
>  is it possible to record nothing on one of the tracks?
 
Yes, it'd be possible. Every recorded video frame has a timestamp associated, if there is a jump in any of the sequences of timestamps, any player would just keep the last drawn canvas, unless, that is, `ignoreMutedMedia` inserts black video frames. 

In the hypothetical case you mention, assuming A and B are video tracks for simplicity, after the `muted` hiatus in track B, they'll still be in sync, because the UA will still produce correct timestamps in the container. If you were playing both in parallel, then A will play all the way with no hiccups, whereas B will either be frozen for 10 seconds, or turn black for ten seconds, depending on `ignoreMutedMedia`.

I'm not sure about the extent of `enabled` versus `muted`. In Chrome, `muted` is a reflection of no video passing by the track during a certain period of time; this being affected by `enabled` or not, depends on where the video flow is interrupted (before or after the Track). Maybe @guidou knows more?